mesonbuild/dependencies/misc.py

```diff
@@ -616,3 +640,9 @@ def shaderc_factory(env: 'Environment',
     system_class=OpensslSystemDependency,
     cmake_class=CMakeDependencyFactory('OpenSSL', modules=['OpenSSL::SSL']),
 )
+
+packages['objfw'] = objfw_factory = DependencyFactory(
+    'objfw',
+    [DependencyMethods.CONFIG_TOOL],
+    configtool_class=ObjFWDependencyConfigTool,
+)
```

Comment:
I'm pretty sure that you don't need an `DependencyFactory` here, since there is only one way to find this. I think it's sufficient to have `packages['objfw'] = ObjFWDependencyConfigTool`

Reply:
Done.